.github/workflows/on-pullrequest-dockerfile-ac.yml

```diff
@@ -0,0 +1,95 @@
+
+  on:
+    push:
+      branches:
+        - adminapi23-rc.*
+    workflow_dispatch:
+
+  env:
+    DOCKER_USERNAME: ${{ vars.DOCKER_USERNAME }}
+    DOCKER_HUB_TOKEN: ${{ secrets.DOCKER_HUB_TOKEN }}
+    IMAGE_NAME: ${{ vars.IMAGE_NAME }}
+
+  permissions: read-all
+
+  jobs:
+    docker-analysis:
+      runs-on: ubuntu-latest
+      permissions:
+        security-events: write
+        pull-requests: write
+      strategy:
+        fail-fast: false
+        matrix:
+          dockerfile:
+            [
+              { name: "development", path: "Docker/dev.pgsql.Dockerfile", type: "local" }
+            ]
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+        - name: Set IMAGE_TAG
+          id: set-image-tag
+          run: |
+            if [ "${{ github.event_name }}" == "pull_request" ]; then
+              echo "IMAGE_TAG=${{ github.base_ref }}" >> $GITHUB_ENV
+            else
+              echo "IMAGE_TAG=${{ github.ref_name }}" >> $GITHUB_ENV
+            fi
+
+        - name: Copy application folder to docker context
+          if: ${{ matrix.dockerfile.type == 'local' }}
+          run: |
+            mkdir Docker/Application
+            cp -r ./Application/EdFi.Ods.AdminApi ./Docker/Application
+            cp -r ./Application/EdFi.Ods.AdminApi.AdminConsole ./Docker/Application
+            cp -r ./Application/EdFi.Ods.AdminApi.Common ./Docker/Application
+            cp ./Application/NuGet.Config ./Docker/Application
+
+        - uses: hadolint/hadolint-action@54c9adbab1582c2ef04b2016b760714a4bfde3cf # v3.1.0
+          name: Run Linter on ${{ matrix.dockerfile.name }} Dockerfile
+          with:
+            dockerfile: ${{ matrix.dockerfile.path }}
+            failure-threshold: error
+
+        - name: Log in to Docker Hub
+          uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
+          with:
+            username: ${{ env.DOCKER_USERNAME }}
+            password: ${{ env.DOCKER_HUB_TOKEN }}
+
+        - name: Build
+          run: |
+            path=${{matrix.dockerfile.path}}
+            folder=${path%/*}
+            cd $folder
+            dockerfile=$(echo ${{matrix.dockerfile.path}} | awk -F"/" '{print $NF}')
+
+            docker build -f $dockerfile -t ${{ matrix.dockerfile.name }} --build-arg="VERSION=${{ env.IMAGE_TAG }}" .
+
+        - name: Analyze
+          uses: docker/scout-action@67eb1afe777307506aaecb9acd9a0e0389cb99ae # v1.5.0
+          with:
+            command: cves
+            image: local://${{ matrix.dockerfile.name }}
+            sarif-file: sarif-${{ matrix.dockerfile.name }}.output.json
+            summary: true
+
+        - name: Push Image on Docker Hub
+          run: |
+            docker image tag ${{ matrix.dockerfile.name }} ${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}
+            docker push ${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}
```

Comment:
Definitely want to make sure this does not trigger on pull request. In fact, I don't think we want to push this image here, since that is already handled by `on-prerelease.yml`.

Reply:
I am deleting this file.